var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -367,28 +419,40 @@ def initconfig_mpi_entries(self):
         else:
             entries.append(cmake_cache_option("ENABLE_MPI", False))
 
+        # Replace /usr/bin/srun path with srun flux wrapper path on TOSS 4
+        if "toss_4" in self._get_sys_type(spec):
```

Comment:
Does this mean that axom will be completely broken on any non-LC machine running a scheduler other than slurm?

Reply:
As I understand, this has to do with the default srun of the machine not being the flux wrapped version, which this finds. This is specific to flux machines on LC.

Reply:
Logged an issue #42298